packages/pigeon/example/app/ios/Runner/Messages.g.swift

```diff
@@ -173,8 +173,11 @@ class ExampleHostApiSetup {
     }
   }
 }
-/// Generated class from Pigeon that represents Flutter messages that can be called from Swift.
-class MessageFlutterApi {
+protocol MessageFlutterApiProtocol {
+  func flutterMethod(aString aStringArg: String?, completion: @escaping (Result<String, FlutterError>) -> Void) 
+}
+/// Generated protocol from Pigeon that represents Flutter messages that can be called from Swift.
```

Comment:
Is this supposed to be on the protocol? This class is an implementation, not a protocol.